source/instances/object_map.js

```diff
@@ -0,0 +1,9 @@
+import _reduce from '../internal/_reduce';
+import keys from '../keys';
+
+export default function object_map(f, obj) {
+  return _reduce(function(acc, key) {
```

Comment:
[This test](https://github.com/ramda/ramda/blob/1a61378c1811479d9820f13581345a9baacf4129/test/whereEq.js#L63) fails with simple `for (k in obj)` loop, due to the keys iterated over vs what we get with `keys`.